dask/dataframe/multi.py

```diff
@@ -902,6 +902,19 @@ def merge_asof(
             " 'forward', or 'nearest'"
         )
 
+    if left is None or right is None:
+        raise ValueError("Cannot merge_asof on None")
```

Comment:
I realize these lines aren't new, but is it common to have a specific error message for missing positional arguments?

Reply:
No, Python itself will throw an error if there's a positional argument missing. This is checking for `dd.merge_asof(df, None)`, which seems very strange but I didn't want to remove it without more context.